gradle.properties

```diff
@@ -1,12 +1,13 @@
 # Done to increase the memory available to gradle.
 org.gradle.jvmargs=-Xmx1G
 
-enigma_version=1.4.2
+enigma_version=1.4.5
 stitch_version=0.6.1
 unpick_version=2.2.0
-cfr_version=0.0.6
+cfr_version=0.0.7
```

Comment:
I recall the newer cfr has problem with outer class record desugaring (not dumping with the record class dumper). Is that the case in enigma?

Reply:
I havent noticed that, I hope this should be better than what we had before, im sure more fixes will be needed.

Reply:
if that becomes a proble.m, shouldn't be too hard to find a fix either